netcore/System.Private.CoreLib/src/System/Object.cs

```diff
@@ -15,7 +16,7 @@ partial class Object
 		protected extern object MemberwiseClone ();
 
 		[Intrinsic]
-		internal ref byte GetRawData () => throw new NotImplementedException ();
+		internal ref byte GetRawData () => ref Unsafe.As<Mono.RawData>(this).Data;
```

Comment:
Not in favour of this. We have Intrinsic why to have managed code and dependencies which will never be used?

Reply:
The idea was to remove the intrinsic in favour of the managed code. It seems to produce the same codegen.

Reply:
I don't think it's worth it, as we already have the instrincis and the managed code is not really improving the readability or performance.

Reply:
I beg to differ on the readability but I am fine with closing this as the improvement is questionable.

Reply:
Also, we can shared the implementation with CoreCLR which uses basically the same code as I used here.

Reply:
Is the interpreter keeping up with the intrinsics?

Reply:
For this particular case the interpreter has both `object.GetRawData` (netcore only) and `Unsafe.As`.

Reply:
I don't mean performance, but in recognition.
Having the IL loosens the requirement of keeping up with recognition.